src/FsAutoComplete/LspHelpers.fs

```diff
@@ -398,6 +398,50 @@ module Structure =
           EndLine          = lsp.End.Line
           Kind             = kind }
 
+module ClassificationUtils =
```

Comment:
this move isn't strictly necessary, I should move it back. I kinda want to kill the `CommandResponse` module over time though :D

Reply:
I reverted this change to minimize API diffs